src/main/g8/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   akka-serverless-proxy:
-    image: gcr.io/akkaserverless-public/akkaserverless-proxy:0.7.0-beta.19
+    image: gcr.io/akkaserverless-public/akkaserverless-proxy:0.8.3
```

Comment:
Good catch!

Shame we need to update this manually, but I guess this is 'temporary' until we build something into akkasls.

Reply:
added https://github.com/lightbend/akkaserverless-framework/pull/1000